hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/security/ssl/KeyStoreTestUtil.java

```diff
@@ -136,9 +138,14 @@ public static X509Certificate generateSignedCertificate(String dn, KeyPair pair,
       JcaX509ExtensionUtils extUtil = new JcaX509ExtensionUtils();
       ContentSigner sigGen = new JcaContentSignerBuilder(algorithm).setProvider("BC").build(caKey);
       X509v3CertificateBuilder certGen = new JcaX509v3CertificateBuilder(issuer, sn, from, to, x500Name, pair.getPublic())
-          .addExtension(Extension.authorityKeyIdentifier, false, extUtil.createAuthorityKeyIdentifier(caCert
-              .getPublicKey()))
-          .addExtension(Extension.subjectKeyIdentifier, false, extUtil.createSubjectKeyIdentifier(pair.getPublic()));
+              .addExtension(Extension.authorityKeyIdentifier, false, extUtil.createAuthorityKeyIdentifier(caCert
```

Comment:
Why there are such changes here?

Reply:
I think last time when I was releasing 3.2.0.14-SNAPSHOT I created the 3.2.0.14 branches by mistake. that is why it seems a bit odd this time.